sdk/containerregistry/azure-containers-containerregistry/src/test/resources/session-records/ContainerRegistryClientAsyncTest.listRepositoriesWithPageSize[1].json

```diff
@@ -1,159 +1,4 @@
-      "x-ms-client-request-id" : "6ced16db-17fd-4635-ad55-4405cc37811d"
-    },
-    "Response" : {
-      "Server" : "openresty",
-      "X-Content-Type-Options" : "nosniff",
-      "Connection" : "keep-alive",
-      "retry-after" : "0",
-      "StatusCode" : "200",
-      "Date" : "Fri, 26 Mar 2021 00:00:57 GMT",
-      "Docker-Distribution-Api-Version" : "registry/2.0",
-      "X-Ms-Correlation-Request-Id" : "21374ac6-e144-458c-a246-3ba2c90015d3",
-      "Access-Control-Expose-Headers" : "Docker-Content-Digest,WWW-Authenticate,Link,X-Ms-Correlation-Request-Id",
-      "Strict-Transport-Security" : "max-age=31536000; includeSubDomains,max-age=31536000; includeSubDomains",
-      "Content-Length" : "43",
-      "Body" : "{\"repositories\":[\"library/hello-seattle\"]}\n",
-      "Link" : "</acr/v1/_catalog?last=library%2Fhello-seattle&n=1&orderby=>; rel=\"next\"",
-      "Content-Type" : "application/json; charset=utf-8"
-    },
-    "Exception" : null
-  }, {
-    "Method" : "POST",
-    "Uri" : "https://REDACTED.azurecr.io/oauth2/token",
-    "Headers" : {
-      "User-Agent" : "azsdk-java-name/version (11.0.10; Windows 10; 10.0)",
-      "x-ms-client-request-id" : "c2ec564e-2cf1-41a4-832b-3a6aeee338f2",
-      "Content-Type" : "application/x-www-form-urlencoded"
-    },
-    "Response" : {
-      "Transfer-Encoding" : "chunked",
-      "X-Ms-Correlation-Request-Id" : "cd7a3777-f329-4821-aa03-0a567bb95efd",
-      "Strict-Transport-Security" : "max-age=31536000; includeSubDomains",
-      "Server" : "openresty",
-      "Connection" : "keep-alive",
-      "retry-after" : "0",
-      "StatusCode" : "200",
-      "Body" : "{\"access_token\":\"REDACTED\"}",
-      "Date" : "Fri, 26 Mar 2021 00:00:57 GMT",
-      "Content-Type" : "application/json; charset=utf-8"
-    },
-    "Exception" : null
-  }, {
-    "Method" : "GET",
-    "Uri" : "https://REDACTED.azurecr.io/acr/v1/_catalog?last=library%2Fhello-seattle&n=1&orderby=",
-    "Headers" : {
-      "User-Agent" : "azsdk-java-name/version (11.0.10; Windows 10; 10.0)",
-      "x-ms-client-request-id" : "80b6b0ee-f58a-4552-9c64-31ae611ed458"
-    },
-    "Response" : {
-      "Server" : "openresty",
-      "X-Content-Type-Options" : "nosniff",
-      "Connection" : "keep-alive",
-      "retry-after" : "0",
-      "StatusCode" : "200",
-      "Date" : "Fri, 26 Mar 2021 00:00:58 GMT",
-      "Docker-Distribution-Api-Version" : "registry/2.0",
-      "X-Ms-Correlation-Request-Id" : "6d28afe9-3e85-485d-95c5-cb870ec42022",
-      "Access-Control-Expose-Headers" : "Docker-Content-Digest,WWW-Authenticate,Link,X-Ms-Correlation-Request-Id",
-      "Strict-Transport-Security" : "max-age=31536000; includeSubDomains,max-age=31536000; includeSubDomains",
-      "Content-Length" : "41",
-      "Body" : "{\"repositories\":[\"library/hello-world\"]}\n",
-      "Content-Type" : "application/json; charset=utf-8"
-    },
-    "Exception" : null
-  } ],
+  "networkCallRecords" : [ ],
```

Comment:
Something seems off here

Reply:
I can regenerate the records and see what is going on.

---
In reply to: [609169570](https://github.com/Azure/azure-sdk-for-java/pull/20368#discussion_r609169570) [](ancestors = 609169570)